int16 wheel1
int16 wheel2
